### DISCUSSION TITLE
The Challenges For Full-Stack Developers

### DISCUSSION THREAD

**Discussion by MkDev11** [community/community#67417] [Programming Help]
### Select Topic Area

General

### Body

As a **frontend developer**, I have been thinking about the **challenges** that frontend developers may encounter while developing.
There are so many opinions but I don't exactly know about what is the most difficult one and the easiest one.
In a word, I couldn't list the challenges **from easy one to difficult one**, so I decided to discuss it here with everyone.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @MkDev11 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
